<?xml version="1.0" encoding="utf-8"?>
<launch>

    <arg name="sim" default="false"/>
    <arg name="frame_rate" default="5"/>

    <node name="pass_gate" pkg="robotx" type="pass_gate.py"  output="screen" clear_params="true" required="true"/>

    <group if="$(arg sim)">
        <remap from="task5/image_compressed" to="/zed_mid/rgb/image_rect_color/compressed"/>
        <node name="task5" pkg="robotx" type="scan_the_code.py"  output="screen" clear_params="true" required="true">
            <param name="sim" value="$(arg sim)"/>
            <param name="frame_rate" value="$(arg frame_rate)"/>
        </node>
    </group>
    <group unless="$(arg sim)">
        <remap from="task5/image_compressed" to="/zed_mid/rgb/image_rect_color/compressed"/>
        <node name="task5" pkg="robotx" type="scan_the_code.py"  output="screen" clear_params="true" required="true">
            <param name="sim" value="$(arg sim)"/>
            <param name="frame_rate" value="$(arg frame_rate)"/>
        </node>
    </group>

</launch>
